<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Backpack</class>
 <widget class="QWidget" name="Backpack">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>540</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>汉仪文黑-85W</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="bg">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>960</width>
     <height>540</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
border-image: url(:/BG/backpack.png)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="close">
   <property name="geometry">
    <rect>
     <x>900</x>
     <y>3</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: transparent;
border-radius:31px;
border-image: url(:/Icon/close.png)
}

QPushButton:pressed
{
border-image: url(:/Icon/close_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="deleted">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>490</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;删除圣遗物&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: transparent;
border-radius:31px;
border-image: url(:/Icon/delete.png)
}

QPushButton:pressed
{
border-image: url(:/Icon/delete_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="modify">
   <property name="geometry">
    <rect>
     <x>780</x>
     <y>490</y>
     <width>125</width>
     <height>40</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;修改所选圣遗物属性&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: transparent;
border-radius:31px;
border-image: url(:/Icon/modify.png)
}

QPushButton:pressed
{
border-image: url(:/Icon/modify_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="filter">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>490</y>
     <width>42</width>
     <height>42</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;按套装筛选&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton
{
background-color: transparent;
border-radius:31px;
border-image: url(:/Icon/filter.png)
}

QPushButton:pressed
{
border-image: url(:/Icon/filter_selected.png)
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QScrollArea" name="detailArea">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>60</y>
     <width>250</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="detail">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>250</width>
      <height>420</height>
     </rect>
    </property>
    <widget class="QLabel" name="detail_bg1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>246</width>
       <height>174</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Icon/detail5_bg1.png)</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="name">
     <property name="geometry">
      <rect>
       <x>14</x>
       <y>3</y>
       <width>220</width>
       <height>24</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="position">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>33</y>
       <width>35</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="ma">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>75</y>
       <width>120</width>
       <height>17</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #c0afa8;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="mav">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>90</y>
       <width>150</width>
       <height>28</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="artPic">
     <property name="geometry">
      <rect>
       <x>129</x>
       <y>33</y>
       <width>105</width>
       <height>98</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="level">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>154</y>
       <width>32</width>
       <height>15</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="sa1">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>177</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4d5769;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sa2">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>198</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4d5769;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sa3">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>219</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4d5769;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="sa4">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>240</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #4d5769;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="setName">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>261</y>
       <width>200</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #5cb256;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="twoPieceSet">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>282</y>
       <width>218</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>218</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>218</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #6f7581;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="fourPieceSet">
     <property name="geometry">
      <rect>
       <x>13</x>
       <y>303</y>
       <width>218</width>
       <height>21</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>218</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>218</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>汉仪文黑-85W</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">color: #6f7581;
background-color: transparent;</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="detail_bg2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>174</y>
       <width>246</width>
       <height>174</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Icon/detail5_bg2.png);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="detail_bg3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>348</y>
       <width>246</width>
       <height>3</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">border-image: url(:/Icon/detail5_bg3.png);</string>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Plain</enum>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QScrollBar" name="verticalScrollBar_2">
     <property name="geometry">
      <rect>
       <x>246</x>
       <y>0</y>
       <width>4</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置垂直滚动条基本样式*/
QScrollBar:vertical
{
    width:4px;
    background:#515151;
    margin:0px,0px,0px,0px;
    padding-top:4px;   /* 留出9px给上面和下面的箭头*/
    padding-bottom:4px;
}
QScrollBar::handle:vertical
{
    width:4px;
    background:white;
    border-radius:4px;   /* 滚动条两端变成椭圆 */
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:4px;
    background:white;   /* 鼠标放到滚动条上的时候，颜色变深*/
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical   /* 这个应该是设置下箭头的，3.png就是箭头*/
{
    height:4px;width:6px;
    background-color: transparent;
    border-image:url(:/Icon/down.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical   /* 设置上箭头*/
{
    height:4px;width:6px;
	background-color: transparent;
    border-image:url(:/Icon/up.png);
    subcontrol-position:top;
}
QScrollBar::add-line:vertical:hover   /* 当鼠标放到下箭头上的时候*/
{
    height:4px;width:6px;
    background-color: transparent;
    border-image:url(:/Icon/down_selected.png);
    subcontrol-position:bottom;
}
QScrollBar::sub-line:vertical:hover  /* 当鼠标放到下箭头上的时候*/
{
    height:4px;width:6px;
	background-color: transparent;
    border-image:url(:/Icon/up_selected.png);
    subcontrol-position:top;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <zorder>detail_bg2</zorder>
    <zorder>detail_bg1</zorder>
    <zorder>name</zorder>
    <zorder>position</zorder>
    <zorder>ma</zorder>
    <zorder>mav</zorder>
    <zorder>artPic</zorder>
    <zorder>level</zorder>
    <zorder>sa1</zorder>
    <zorder>sa2</zorder>
    <zorder>sa3</zorder>
    <zorder>sa4</zorder>
    <zorder>setName</zorder>
    <zorder>twoPieceSet</zorder>
    <zorder>fourPieceSet</zorder>
    <zorder>detail_bg3</zorder>
    <zorder>verticalScrollBar_2</zorder>
   </widget>
  </widget>
  <widget class="QScrollArea" name="artifactArea">
   <property name="geometry">
    <rect>
     <x>36</x>
     <y>60</y>
     <width>610</width>
     <height>420</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::NoFrame</enum>
   </property>
   <property name="horizontalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOff</enum>
   </property>
   <property name="widgetResizable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="artifact">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>610</width>
      <height>420</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <widget class="QScrollBar" name="verticalScrollBar">
     <property name="geometry">
      <rect>
       <x>603</x>
       <y>0</y>
       <width>4</width>
       <height>420</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">/* 设置垂直滚动条基本样式*/
QScrollBar:vertical
{
    width:4px;
    background:#515151;
    margin:0px,0px,0px,0px;
    padding-top:4px;   /* 留出9px给上面和下面的箭头*/
    padding-bottom:4px;
}
QScrollBar::handle:vertical
{
    width:4px;
    background:white;
    border-radius:4px;   /* 滚动条两端变成椭圆 */
    min-height:20;
}
QScrollBar::handle:vertical:hover
{
    width:4px;
    background:white;   /* 鼠标放到滚动条上的时候，颜色变深*/
    border-radius:4px;
    min-height:20;
}
QScrollBar::add-line:vertical   /* 这个应该是设置下箭头的，3.png就是箭头*/
{
    border:none;
}
QScrollBar::sub-line:vertical   /* 设置上箭头*/
{
    border:none;
}</string>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="text">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>9</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>汉仪文黑-85W</family>
     <pointsize>10</pointsize>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: transparent;
color: rgb(211, 188, 142);</string>
   </property>
   <property name="text">
    <string>圣遗物</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
